### DISCUSSION TITLE
How to Pause a job

### DISCUSSION THREAD

**Discussion by adarshasp87** [jobrunr/jobrunr#209] [Q&A] [answered]
I want to know is there a way to pause or suspend a job?

**Comment by rdehuyss** [comment, marked as answer]
> I want to know is there a way to pause or suspend a job?

A recurring job or a job that is already enqueued?

**Reply by adarshasp87** [reply]
Looking for an option to pause recurring job.

**Reply by rdehuyss** [reply]
This is a planned version for the pro version.

**Reply by rdehuyss** [reply]
FYI - will be part of JobRunr Pro 4
